C#/C#.sublime-syntax

```diff
@@ -192,8 +194,14 @@ contexts:
         1: keyword.other.preprocessor.cs
         2: keyword.other.preprocessor.cs
         3: keyword.other.preprocessor.cs
-    - match: .*
-      scope: invalid.illegal.cs
+
+    - match: '!'
+      scope: punctuation.definition.comment.cs
+      set: shebang-body
```

Comment:
This pattern causes #! being scoped as 2 tokens is not ideal, even if it visually looks as one.

The main purpose of `shebang` context is to only scope first line as such. With this pattern scoping it everywhere `shebang` was redundant.

If shebang is not supported in the middle of files we could probably accept test assertions not being highlighted or just use a dedicated a dedicated `syntax_test_shebang.cs` for shebang tests, while keeping test assertions in normal comment style for `#:`.

Reply:
`#`in this case represents preprocessor directives, not comments. I checked and `dotnet run` throws an error if the shebang is not at the first line of the file, so I removed this part. See what you think of the latest changes :)